.travis.yml

```diff
@@ -5,7 +5,9 @@ before_script:
   - mysql -e 'create database double_entry_test;'
   - psql -c 'create database double_entry_test;' -U postgres
 script:
-  - bundle exec rake spec
+  - bundle exec rspec spec/double_entry spec/active_record
+  - bundle exec rspec spec/generators
+  - bundle exec rspec spec/performance
```

Comment:
Encountered a few spec ordering issues on the `sqlite` test run. It seems the generator specs cause issues with the database file causing it to become read-only. Resolved by running generator specs in a separate process.

Reply:
Hmmm... I am a little bit worried that this could be a pitfall if we add any new directories under `spec/`. 🤔

Reply:
That's a good point.

I'll investigate further the reason for `generators` specs interfering with the other unit tests. If easy to fix, revert this change.